api/api_ethereum.go

```diff
@@ -616,6 +617,15 @@ func (api *EthereumAPI) GetBlockTransactionCountByHash(ctx context.Context, bloc
 	return transactionCount
 }
 
+// accessListResult returns an optional accesslist
+// Its the result of the `debug_createAccessList` RPC call.
+// It contains an error if the transaction itself failed.
+type accessListResult struct {
```

Comment:
I thought you applied latest dev...
Can you check why still this is added in this PR?

Reply:
Seems like there was some issue when I resolved conflicts.. 😭 
I removed it